lib/presentation/localizations/app_localizations.dart

```diff
@@ -1482,6 +1482,14 @@ class AppLocalizations {
     return Intl.message('Biometric authentication is locked due to exceeded number of attempts.',
         name: 'biometric_authentication_is_locked');
   }
+
+  String touch_id_or_face_id(String touchID, String faceID) {
```

Comment:
Why do you add parameters to this just write a raw string so we can translate it IMO

Reply:
Because depending on each platform, there are different biometric styles. With Android it's Face / Fingerprint and iOS it's TouchID / FaceID. So instead of creating many different documents. I just need to translate those biometric keywords. Do you have any better ideas?

Reply:
What do you mean by many different documents ?

I think you should just create 2 strings face_or_fingerprint and touch_id_or_face_id